cmd/manager/main.go

```diff
@@ -110,17 +111,6 @@ func run() error {
 		return err
 	}
 
-	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
```

Comment:
just a comment: seems like we're still exposing the metrics at http://localhost:8383/metrics
I assume they are being exposed by default...